_chapters/chapters/sys-lit-review.md

```diff
@@ -1,7 +1,19 @@
-# Systematic Literature Review
+---
+layout: page 
+title: Systematic Literature Review
+hide: true 
+permalink: /chapter/sys-lit-review
```

Comment:
Need to follow the instruction in the template to make the permalink.
### Permalink 
Each chapter should be given a permalink which serves as an identifier for your chapter. Permalinks define the path on which the chapter will be found on the website **and can be different from its path in the repository**. This facilitates structuring your content. This permalink is crutial for linking your chapter in the table of content and should be of the form `/chapter/[topic name]/[chapter title]/` where `[topic name]` should be used to refer to the topic area of your chapter and `[chapter title]` is used to identify the chapter based in its title.	Each chapter should be given a permalink which serves as an identifier for your chapter. Permalinks define the path on which the chapter will be found on the website **and can be different from its path in the repository**. This facilitates structuring your content. This permalink is crucial for linking your chapter in the table of content and should be of the form `/chapter/[topic name]/[chapter title]/` where `[topic name]` should be used to refer to the topic area of your chapter and `[chapter title]` is used to identify the chapter based in its title.